Advanced_Projects/PySleuth/core/explainer.py

```
@@ -0,0 +1,36 @@
+import ast
+from core.ast_parser import ASTParser
```

Ruff (0.12.2)

`core.ast_parser.ASTParser` imported but unused

Remove unused import: `core.ast_parser.ASTParser`

(F401)